core/src/main/java/org/radargun/stages/test/legacy/LegacyStressor.java

```diff
@@ -87,15 +91,18 @@ private void runInternal() {
                if (ongoingTx != null) {
                   endTransactionAndRegisterStats(null);
                }
-               stats.begin();
-               this.started = started;
                break;
-            }
-            Operation operation = operationSelector.next(random);
-            try {
-               logic.run(operation);
-            } catch (OperationLogic.RequestException e) {
-               // the exception was already logged in makeRequest
+            } else {
```

Comment:
Hey @rvansa, I have made the changes, but I am still curious why it matters that the threads execute operations that won't be recorded while the stage is starting up. What is the purpose of that behavior?